cypress/e2e/challenges.cy.js

```diff
@@ -0,0 +1,70 @@
+import ChallengesPage from '../pages/challengesPage'
+const challengesPage = new ChallengesPage()
+
+describe('example to-do app', () => {
+  beforeEach(() => {
+    cy.visit('localhost:8080')
+  })
+
+  it('Check all challenges load correctly', () => {
+    cy.wrap([0, 1, 2, 3, 4, 8, 12, 13, 14, 15, 16, 17, 18, 19]).each((challengeNum) => {
```

Comment:
is there a way to automatically check for all challenges? (E.g. read a var/indicator at the home page)

Reply:
Yeah thats one of my next challenges. Perhaps from the adocs. I think it wouod be best if new challenges dont need any new UI tests

Reply:
I would consider addding an endpoint which retuns the number of challenges. This might also useful in the FE itself. Could be an actuator endpoint with some meta information.

Reply:
You can actually use https://wrongsecrets.herokuapp.com/api/challenges (e.g. localhost:8080/api/challenges) to get all the challenges and see if they are enabled or not

Reply:
Check https://wrongsecrets-commjoen.cloud.okteto.net/swagger-ui/index.html#/challenges-api-controller/getChallenges :)

Reply:
Brilliant. That will work perfectly. Ill give it a go tonight

Reply:
The challenges are now resolved using this endpoint :)